### PR TITLE
Update youtube.spec.ts

### PR DESCRIPTION
**Pull request by peter9811**

### CODE REVIEW

**Comment by alexesprit** [comment]
It's better to add tests for a feature in the same PR. But we'll keep this one.

**Comment by alexesprit** [comment]
By the way, the tests won't pass because you have no changes in `youtube` function in this branch.

**Comment by peter9811** [comment]
Aparently do it, but need help to do the rebase...
Or I can delete the fork and do it again & copy&paste the changes that I realized?

File: test/function/youtube.spec.ts [hunk 1]
```diff
@@ -343,4 +343,25 @@ testFilterFunction(youtube, [
...
+	{
+		description: 'should remove "(Letra)" string',
+		funcParameter: 'Track Title [Letra]',
+		expectedValue: 'Track Title',
+	},
+	{
+		description: 'should remove "(Lyrics) + [Lyrics]" string',
+		funcParameter: 'Track Title (Lyrics) [Lyrics]',
+		expectedValue: 'Track Title',
+	},
```
Comment (alexesprit) [inline test/function/youtube.spec.ts:366]: What is a purpose of these tests? These cases are already covered.

Comment (peter9811) [reply test/function/youtube.spec.ts:366]: Oh... My fault, sorry 
I was checking when o Youtube the people use the two languages

**Pull request closed without merging**